### PR TITLE
Show snackbar message for developers when hotreloading

### PR DESCRIPTION
**Pull request by lukaw3d**

### CODE REVIEW

**Comment by Midnighter** [comment]
`module.hot` only exists/is only defined in development?

**Comment by lukaw3d** [comment]
Yes, webpack-dev-server

File: src/App.vue [hunk 1]
```diff
@@ -255,7 +259,8 @@ export default Vue.extend({
     drawer: false,
     isProjectCreationDialogVisible: false,
     isOrganismCreationDialogVisible: false,
-    isExpanded: true
+    isExpanded: true,
+    hotreloadStatus: undefined
```
Comment (kvikshaug) [inline src/App.vue]: I don't know if it makes much of a difference but at least in the past we tried to prefer `null` over `undefined` consistently where possible.

Comment (lukaw3d) [reply src/App.vue]: Oh, right :+1:

**Pull request merged**